frontend/src/requests/FinnHub-Requests.js

```diff
@@ -0,0 +1,49 @@
+
+export function stockPricesRequest(symbol, range, start, end, setFunction) {
+    const finnhub = require('finnhub');
+    const api_key = finnhub.ApiClient.instance.authentications['api_key'];
+    api_key.apiKey = process.env.FINNHUB_API_KEY;
+    const finnhubClient = new finnhub.DefaultApi()
+    finnhubClient.stockCandles(symbol, range, start, end, async (error, data, response) => {
+        if (error === null) setFunction(data);
+        else console.log('error', error);
+    });
+};
+
+
+export function stockSearch(symbol, setFunction) {
+    const finnhub = require('finnhub');
+    const api_key = finnhub.ApiClient.instance.authentications['api_key'];
+    api_key.apiKey = process.env.FINNHUB_API_KEY;
+    const finnhubClient = new finnhub.DefaultApi()
+    finnhubClient.symbolSearch(symbol, (error, data, response) => {
+        if (error === null) setFunction(data);
+        else console.log('error', error);
+    });
+};
+
+export function marketNews(category, setFunction) {
+    const categories = ['general', 'forex', 'crypto', 'merger']
+    const finnhub = require('finnhub');
+    const api_key = finnhub.ApiClient.instance.authentications['api_key'];
+    api_key.apiKey = process.env.FINNHUB_API_KEY;
+    const finnhubClient = new finnhub.DefaultApi()
+    finnhubClient.marketNews(category in categories ? category : 'general', {}, (error, data, response) => {
```

Comment:
I don't know the use of the second parameter of the Finnhub marketNews function, but we may find it useful in the future. It could be passed as a parameter in our marketNews function, and then passed to the Finnhub function, just in case. When that parameter isn't desired an empty object can be passed through the parameters. It should work as expected.